packages/taxdat/R/setup_helpers.R

```diff
@@ -454,6 +454,10 @@ check_set_tfrac <- function(set_tfrac) {
 #'
 check_snap_tol <- function(snap_tol, res_time) {
```

Comment:
@Qulu it occurs to me that we should add a few unit test for check_snap_tol
* does the error get returned when snap_tol is not evaluable?
* does the correct numeric value get returned when snap_tol is a string that is evaluable?